packages/ui-components/.storybook/webpack.config.js

```diff
@@ -36,6 +36,9 @@ module.exports = async ({ config }) => {
     ...config.resolve.alias,
     yargs: path.resolve(__dirname, 'moduleMock.js'),
     child_process: path.resolve(__dirname, 'moduleMock.js'),
+    '@aws-sdk/credential-providers': path.resolve(__dirname, 'moduleMock.js'),
```

Comment:
Is this the same issue as RN-950?

Reply:
I think so! It just started so I think it must be